pkg/noderesourcetopology/cache/overreserve.go

```diff
@@ -206,6 +213,13 @@ func (ov *OverReserve) Resync() {
 			continue
 		}
 
+		objs, ok := nodeToObjsMap[nodeName]
+		if !ok || len(objs) == 0 {
```

Comment:
`if !ok` shouldn't be enough?

Reply:
yes, and more idiomatic. I'll fix in the next upload.